Configuration/AlCa/python/autoCond.py

```diff
@@ -69,17 +69,17 @@
     # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in PEAK mode
     'phase1_2018_cosmics_peak' :  '112X_upgrade2018cosmics_realistic_peak_v3',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2021
-    'phase1_2021_design'       : '112X_mcRun3_2021_design_v7', # GT containing design conditions for Phase1 2021
+    'phase1_2021_design'       : '112X_mcRun3_2021_design_v8', # GT containing design conditions for Phase1 2021
     # GlobalTag for MC production with realistic conditions for Phase1 2021
-    'phase1_2021_realistic'    : '112X_mcRun3_2021_realistic_v7', # GT containing realistic conditions for Phase1 2021
+    'phase1_2021_realistic'    : '112X_mcRun3_2021_realistic_v8', # GT containing realistic conditions for Phase1 2021
```

Comment:
I was expecting to see also a new `ExtentedZeroMaterial` labeled tag for `GeometryFileRcd`, based on the `Extended2021ZeroMaterial` scenario introduced in https://github.com/cms-sw/cmssw/pull/31082. Will that follow in another PR?

Reply:
@cvuosalo - IMHO, these scenarios should be uploaded together. Otherwise, we should request to delete `ExtentedZeroMaterial` from this GT.

Reply:
at the moment there is no such label in that GT:
```
$ conddb list 112X_mcRun3_2021_realistic_v8 | grep GeometryFileRcd
[2020-09-22 12:23:13,561] INFO: Connecting to pro [frontier://PromptProd/CMS_CONDITIONS]
GeometryFileRcd                                         CTPPS                                                          XMLFILE_CTPPS_Geometry_2018_101YV6                                                  
GeometryFileRcd                                         Extended                                                       XMLFILE_Geometry_112YV2_Extended2021_mc                                             
MFGeometryFileRcd                                       120812                                                         MFGeometry_120812                                                                   
MFGeometryFileRcd                                       130503                                                         MFGeometry_130503                                                                   
MFGeometryFileRcd                                       160812                                                         MFGeometry_160812                                                                   
MFGeometryFileRcd                                       90322                                                          MFGeometry_90322         
```

Reply:
Does a tag for the `ExtendedZeroMaterial` scenario exist? I understood from @cvuosalo comments at yesterday's AlCaDB meeting that it did not and that we should not wait for the zero material scenario to converge. Can the payload corresponding to #31082 be generated now or is additional work required?

@cvuosalo @ianna Please also see and respond to @fwyzard's [recent HN post ](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/4338/2/1.html) regarding use of old MC samples in releases built after this PR is merged.

Reply:
>  I understood from @cvuosalo comments at yesterday's AlCaDB meeting that it did not and that we should not wait for the zero material scenario to converge.

As far as I can see there are some samples for EGM that need that to fall in place in order to be injected [*], so some priority would be needed. Alternatively the driver command for those samples could be tweaked in order to deliver the simulation geometry from XML.

[*] https://hypernews.cern.ch/HyperNews/CMS/get/prep-ops/7302.html

Reply:
> Does a tag for the `ExtendedZeroMaterial` scenario exist? I understood from @cvuosalo comments at yesterday's AlCaDB meeting that it did not and that we should not wait for the zero material scenario to converge. Can the payload corresponding to #31082 be generated now or is additional work required?
> 
> @cvuosalo @ianna Please also see and respond to @fwyzard's [recent HN post ](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/4338/2/1.html) regarding use of old MC samples in releases built after this PR is merged.

I'll check with @cvuosalo about the Zero material payload. The scripts used to produce all Sim geometry variations at once.

I've noticed there is a `GeometryFileRcd`  with a `CTPPS` label. Is it with iovs? e.g. is it their Reco geometry? It should eventually go away, I think, they have decoupled their Reco geometry from the XML now.

The last point - the Magnetic field geometry. How old the payloads are? The change in Muon yoke thickness should be identical for both Sim geometry and MF geometry.

Sorry, I would have checked it myself, but I'm having trouble connecting to CondDB...

Reply:
@tsusa Are the changes in #31082 sufficient to properly define the zero material scenario for Run 3? I (mis?)understood from your comments at yesterday's AlCa meeting that additional work was required.

Reply:
The real changes proposed in #31082 involve only the densities of the non-active volumes and IIUC are not coupled with the other (small) tracker changes (solution of overlaps, etc.) for Run-3. I would say the payload could be derived right away